src/lib/FileReadDataSource.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
```

Comment:
This file should be under `./datasource` folder

Reply:
done